tests/bad-proofs/mod-on-negative-spec.k

```diff
@@ -0,0 +1,10 @@
+requires "kwasm-lemmas.k"
+
+module MOD-ON-NEGATIVE-SPEC
+    imports KWASM-LEMMAS
+
+    rule <k> X modInt N => X ... </k>
+      requires X <=Int 0
+       andBool 0  <Int N
+
+endmodule
```

Comment:
Would recommend taking at least one semantic step in all proofs to avoid weird corner cases in the java backend.

Something as simple as:

```
syntax Stmt ::= doStep ( Int )
rule <k> doStep ( I ) => I ... </k>
```

Reply:
Trying to get this to work ...

Added  to test.md:
```
    syntax KItem ::= "doStep" "(" Int ")" | "doneStep" "(" Int ")"
 // --------------------------------------------------------------
    rule <k> doStep ( I ) => doneStep ( I ) ... </k>
```

and the proof spec is now:

```
    rule <k> doStep ( X modInt N ) => doneStep ( X ) ... </k>
      requires X <=Int 0
       andBool 0  <Int N
```

The prover fails, seemingly because it fails to match on the initial configuration, ` Subject: doStep(_modInt_(X_918:Int,, N_917:Int))`